### PR TITLE
Delete your Status Community

### PR DESCRIPTION
**Pull request by cheny0**

### CODE REVIEW

File: docs/en/status-communities/delete-your-status-community.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+!!! note
+    Your messages are not in the blockchain and are not transported through the Ethereum network. Messages are temporarily stored in the peer-to-peer network.
+
+## Delete your Status Community
+
+You must complete the following tasks:
+
+- [x] Require [manual approval][manage-community-join-requests] for all join requests.
+- [x] [Kick all members from the community][kick-or-ban-someone-from-your-community].
```
Comment (jorge-campo) [inline docs/en/status-communities/delete-your-status-community.md:29]:
```suggestion
- [x] [Kick out all members from the community][kick-or-ban-someone-from-your-community].
```

File: docs/en/status-communities/delete-your-status-community.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- [x] [Leave your community][leave-a-status-community].
+- [x] Delete and destroy the community private key if it's stored anywhere.
+- [x] Wait 30 days.
+
```
Comment (jorge-campo) [inline docs/en/status-communities/delete-your-status-community.md:33]:
```suggestion

After 30 days, the Waku peer-to-peer network stops caching your community data, and the community is deleted irrecoverably.

```

**Review by jorge-campo — CHANGES_REQUESTED** [review]
And the `index.md` entry ? :)

Thanks for the article, @cheny0 - Simple and to-the-point. 

I only have a few suggestions that I think can improve the article's readability.

**Comment by cheny0** [comment]
> And the `index.md` entry ? :)
> 
> Thanks for the article, @cheny0 - Simple and to-the-point.
> 
> I only have a few suggestions that I think can improve the article's readability.

Thanks! I have applied the changes and merged the article into `develop` and `master`.

**Pull request merged**